pandas/core/series.py

```diff
@@ -4390,7 +4390,7 @@ def to_period(self, freq=None, copy=True):
     ["index"], docs={"index": "The index (axis labels) of the Series."},
 )
 Series._add_numeric_operations()
-Series._add_series_only_operations()
+# Series._add_series_only_operations()
```

Comment:
delete.  basically never leave commented-out code in a PR

Reply:
actually made this commit to get some help on getting this future warning

Reply:
> so I removed code and tests for ptp
> remove the part where series only functions are added
> removed the entire series only function because ptp was the only series only function
> deleted tests for pandas ptp function
> but the test written for np.ptp gives me a future warning
> 
> `
> self = <pandas.tests.series.test_analytics.TestSeriesAnalytics object at 0x7f666e0267b8>
> 
> ```
> def test_ptp(self):
>     # GH21614
>     N = 1000
>     arr = np.random.randn(N)
>     ser = Series(arr)
>     with tm.assert_produces_warning(FutureWarning, check_stacklevel=False):
> ```
> 
> > ```
> >       assert np.ptp(ser) == np.ptp(arr)
> > ```
> 
> pandas/tests/series/test_analytics.py:861:
> `

Reply:
Remove the assert_produces_warning, since it shouldn't anymore.

Reply:
it works but can someone please explain to me the reason behind it so that I learn